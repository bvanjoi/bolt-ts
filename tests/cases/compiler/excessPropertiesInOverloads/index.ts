// From `github.com/microsoft/TypeScript/blob/v5.8.2/tests/cases/compiler/excessPropertiesInOverloads.ts`, Apache-2.0 License

declare function fn(a: { x: string }): void;
declare function fn(a: { y: string }): void;
fn({ z: 3, a: 3 });
//~^ ERROR: No overload matches this call.