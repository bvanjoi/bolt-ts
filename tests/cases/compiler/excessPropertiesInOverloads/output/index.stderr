  × No overload matches this call.
   ╭─[../../tests/cases/compiler/excessPropertiesInOverloads/index.ts:5:1]
 3 │ fn(a: { x: string }): void;
   · ─┬
   ·  ╰── Unmatched call.
 4 │ declare function fn(a: { y: string }): void;
   ·                  ─┬
   ·                   ╰── Unmatched call.
 5 │ fn({ z: 3, a: 3 })
   · ──────────────────
   ╰────
